controllers/users.js

```diff
@@ -133,6 +124,7 @@ const getUsers = async (req, res) => {
 
 const getUser = async (req, res) => {
   try {
+    // replace below line woth call to retrieveUsers(id)
```

Comment:
If its a todo comment then please add a TODO prefix to it and create a new issue for it